src/sentry/management/commands/makemigrations.py

```diff
@@ -48,7 +48,7 @@ def handle(self, *app_labels, **options):
                 "Please name your migrations using `-n <migration_name>`. "
                 "For example, `-n backfill_my_new_table`"
             )
-            return
+            sys.exit(1)
```

Comment:
added this becasue the `makemigrations` was returning successfully for the error state. This change will allow the `make-migration` bin script to correctly short circuit.